<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LobbyView</class>
 <widget class="QWidget" name="LobbyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lobbyIdDescriptionLabel">
       <property name="text">
        <string>lobby ID:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lobbyIdLabel">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="playersLw"/>
   </item>
   <item>
    <widget class="QPushButton" name="startPb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
